docusaurus/docs/welcome/instructions.md

```diff
@@ -11,6 +11,24 @@ Keep in mind, part of adapting this boilerplate into a deliverable project is al
 
 ## 1. Tweak some project files
 
+### 1.1 Environment Configuration (.env)
```

Comment:
I was thinking this through and I believe this whole section can be moved to "What's included > Environment variables". Using environment variables is entirely opcional, therefore it shouldn't be considered as instructions. Do you agree?